backend/services/syson-services/src/main/java/org/eclipse/syson/util/DescriptionNameGenerator.java

```diff
@@ -44,4 +55,29 @@ protected static String getCompartmentItemName(String prefix, String type) {
     protected static String getEdgeName(String prefix, String type) {
         return getName(prefix, "Edge", type);
     }
+
+    public static String getCreationToolName(EClassifier eClassifier) {
+        return getCreationToolName("New ", eClassifier);
+    }
+
+    public static String getCreationToolName(String prefix, EClassifier eClassifier) {
```

Comment:
It seems the name of Compartment Items creation tools are not valid anymore for all XXXDefinition compartments. Could you check on your side and fix if needed please?

Reply:
fixed